components/profile/time.rs

```diff
@@ -263,8 +263,8 @@ impl Profiler {
                     // (using ProfilerChan.send() forces an unwrap and sometimes panics for this background profiler)
                     let ProfilerChan(ref c) = profiler_chan;
                     if let Err(_) = c.send(ProfilerMsg::Time((ProfilerCategory::ApplicationHeartbeat, None),
-                                                   (start_time, end_time),
-                                                   (start_energy, end_energy))) {
+                                           (start_time, end_time),
+                                           (start_energy, end_energy))) {
```

Comment:
Um, still not the right indentation? Those are part of the `ProfilerMsg::Time` variant.